conference/src/main/java/org/spine/samples/lobby/conference/ConferenceService.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2015, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine.samples.lobby.conference;
+
+import org.spine3.samples.lobby.common.ConferenceId;
+import org.spine3.samples.lobby.common.SeatType;
+import org.spine3.samples.lobby.common.SeatTypeId;
+import org.spine3.samples.lobby.conference.ConferenceInfo;
+import org.spine3.samples.lobby.conference.CreateConferenceResponse;
+import org.spine3.samples.lobby.conference.EditableConferenceInfo;
+import org.spine3.samples.lobby.conference.FindConferenceRequest;
+import org.spine3.samples.lobby.conference.PublishConferenceRequest;
+import org.spine3.samples.lobby.conference.PublishConferenceResponse;
+import org.spine3.samples.lobby.conference.UnpublishConferenceRequest;
+import org.spine3.samples.lobby.conference.UnpublishConferenceResponse;
+import org.spine3.samples.lobby.conference.UpdateConferenceResponse;
+import org.spine3.samples.lobby.conference.contracts.Conference;
+
+import java.util.Set;
+
+/**
```

Comment:
`Service`-oriented approach is not the best thing for apps built on top of Spine.

It should be migrated to either

* aggregates (to handle commands such as `createConference`) and projections — in case this service is NOT just an app facade;
* or to `CommandService` / `QueryService`  in case this IS an app facade.

This must be resolved before 0.6 release. It's a blocker issue.

For now, let's add a `todo` item.